### PR TITLE
docs(global):  improve  contributing.md and contributing.zh-CN.md more official

### PR DESCRIPTION
**Pull request by chenying2929**
1. Adjust the language style to make it more formal. The core content is basically unchanged.
2. Adjust the content structure and image placement of **submit issues**.

### CODE REVIEW

File: CONTRIBUTING.zh-CN.md [hunk 1]
```diff
@@ -1,49 +1,48 @@
...
-如果你还没有 GitHub 账号，先 [申请一个](https://github.com/join)。这很值得。
+如您尚未拥有 GitHub 账号，建议您先 [申请一个](https://github.com/join)。这一步骤非常值得。
 
-然后，你可以通过提交 [issue](https://github.com/antvis/AVA/issues) 或 [pull request](https://github.com/antvis/AVA/pulls) 的方式来提供你的建议。
+接下来，你可以通过提交 [issue](https://github.com/antvis/AVA/issues) 或 [pull request](https://github.com/antvis/AVA/pulls) 的方式来提供你的建议。
 
 ### 提交 Issue
 
-GitHub 的 issues 是一个类似评论区的功能，你可以在这里留下你的想法、建议、发现的 bug，等等。
+GitHub 的 issues 类似于评论区，您可以在此处留下有关想法、建议、需求等方面的反馈。此外，您还可以在此发现并报告仓库中的 bug。以下是提交 issues 的基本步骤：
```
Comment (neoddish) [inline CONTRIBUTING.zh-CN.md:20]: Update to Github 'Discussion' now

File: CONTRIBUTING.zh-CN.md [hunk 2]
```diff
@@ -93,12 +91,11 @@ $ git push origin branch-name
...
 #### 代码风格
 
-你的代码风格必须通过 eslint，你可以运行 `$ npm run lint` 本地测试。
+您的的代码风格必须通过 eslint，可以运行 `$ npm run lint` 进行本地测试。
 
 #### Commit 提交规范
 
-根据 [angular 规范](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format)提交 commit，
-这样 history 看起来更加清晰，还可以自动生成 changelog。
+我们建议遵循 [angular 规范](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format) 来提交 commit。这样能够使历史记录更加清晰易懂，并且还可以方便地自动生成 changelog。
```
Comment (neoddish) [inline CONTRIBUTING.zh-CN.md:98]: check again. @pddpd

File: CONTRIBUTING.zh-CN.md (hunk 1 above)
Comment (neoddish) [reply CONTRIBUTING.zh-CN.md:20]: @chenluli

**Review by neoddish — COMMENTED** [review]
@chenluli @pddpd

File: CONTRIBUTING.zh-CN.md (hunk 2 above)
Comment (chenluli) [reply CONTRIBUTING.zh-CN.md:98]: 下面 scope 的范围有更新明确范围，可以按照我们的实际要求更新下

File: CONTRIBUTING.zh-CN.md [hunk 3]
```diff
@@ -57,34 +56,33 @@ GitHub 的 issues 是一个类似评论区的功能，你可以在这里留下
...
 ### 提交代码
 
 #### 提交 Pull Request
 
-如果你有仓库的开发者权限，而且希望贡献代码，那么你可以创建分支修改代码提交 PR，AntV 开发团队会 review 代码合并到主干。
+如果您拥有仓库的开发者权限，并希望向项目中贡献代码，那么您可以创建分支并修改代码，随后提交 pull request。AntV 开发团队会对您提交的代码进行 review，并将其合并到主干分支中。
 
 ```bash
-# 先创建开发分支开发，分支名应该有含义，避免使用 update、tmp 之类的
+# 先创建开发分支开发，分支名应该有含义，避免使用 update、tmp 等
```
Comment (chenluli) [inline CONTRIBUTING.zh-CN.md:71]: 可以加一下开发的步骤 
Steps for development
```
npm run one-stop-setup

// 开发模式下启动官网
npm run start:site

// playground
npm run start:playground
```

File: CONTRIBUTING.zh-CN.md (hunk 1 above)
Comment (chenying2929) [reply CONTRIBUTING.zh-CN.md:20]: done

File: CONTRIBUTING.zh-CN.md (hunk 2 above)
Comment (chenying2929) [reply CONTRIBUTING.zh-CN.md:106]: done~

File: CONTRIBUTING.zh-CN.md (hunk 3 above)
Comment (chenying2929) [reply CONTRIBUTING.zh-CN.md:66]: done~

**Review by chenluli — APPROVED** [review]
LG

**Pull request merged**